.github/CONTRIBUTING.rst

```diff
@@ -26,13 +26,16 @@ land a single large feature.
 
 Checklist:
 
-1. All pull requests *must* be made against the ``master`` branch.
-2. Include tests for any functionality you implement. Contributions that
+1. Always create a new branch to work on a new issue.
+2. Make sure your branch is rebased with upstream master before you file
+   a pull request.
+3. All pull requests *must* be made against the ``master`` branch.
+4. Include tests for any functionality you implement. Contributions that
    improve existing tests are welcome.
-3. Update documentation as necessary and provide documentation for any new
+5. Update documentation as necessary and provide documentation for any new
    functionality.
-4. In case of UI changes, please include screenshots.
-5. Add yourself to ``CONTRIBUTORS.txt`` if you're not there already. :)
+6. In case of UI changes, please include screenshots.
+7. Add yourself to ``CONTRIBUTORS.txt`` if you're not there already. :)
```

Comment:
I couldn't find the file ``CONTRIBUTORS.txt``,  so created an [issue](https://github.com/pythonindia/junction/issues/635) regarding it. Let me know the issue is valid, I could make the changes related to that in this pr only.

Reply:
Yea, go ahead and remove this line.

I plan on revamping the README at a later date.

Reply:
Done.